core/tests/performance_tests/splash_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the splash page"""
+
+import unittest
+from test_utils import TestBase
+
+
+class SplashPageTest(TestBase):
```

Comment:
Maybe to differentiate more from the backend tests, we could call this:

``SplashPagePerformanceTest``

Thoughts?